### PR TITLE
Enhancements in Bank Transfer/Buy Crypto pages

### PR DESCRIPTION
**Pull request by guptar16**
Enhancements in Bank Transfer and Buy Crypto pages.

### CODE REVIEW

**Comment by radek1st** [comment]
@sonu-sonam please have a look if all good and merge

**Comment by radek1st** [comment]
ah, just noticed, this should go to staging first, not master

File: pages/buy-crypto.js [hunk 1]
```diff
@@ -89,7 +89,7 @@ class BuyCryptoForm extends React.Component {
           }) => (
               <form onSubmit={handleSubmit}>
                 <div className="form">
-                  <div>
+                  <div class = "textalignment">
```
Comment (sonu-sonam) [inline pages/buy-crypto.js:92]: should be using className instead of class

File: pages/buy-crypto.js [hunk 2]
```diff
@@ -355,8 +355,8 @@ class BuyCrypto extends React.Component {
                   typeSpeed={100}
                   loop
                 />
-          
-                at best international crypto exchange rates
+               <br></br>
```
Comment (sonu-sonam) [inline pages/buy-crypto.js:358]: "< br/>" should be enough

File: pages/buy-crypto.scss [hunk 3]
```diff
@@ -677,6 +677,17 @@
   }
 }
 
+p.p1{
```
Comment (sonu-sonam) [inline pages/buy-crypto.scss:680]: names of classes doesn't look good and consistent with the rest.

**Comment by radek1st** [comment]
@guptar16 could you please address the comments? thanks

**Comment by radek1st** [comment]
> @radek1st @sonu-sonam , Let me know if any changes are required.

I leave it to you Sonam

**Comment by sonu-sonam** [comment]
SEO enhancements done from another PR - feature/seo_enhancements 
closing this one.

**Pull request closed without merging**